src/opera/test/pge/disp_s1/test_disp_s1_pge.py

```diff
@@ -404,6 +404,158 @@ def test_disp_s1_pge_validate_inputs(self):
         for f in files_to_remove:
             os.remove(f)
 
+    def test_get_cslc_input_burst_id_set(self):
+        """
+        Set of tests to sanity check the burst ids associated with:
+        cslc_input_files, amplitude_dispersion_files, amplitude_mean_files,
+        and geometry_files.
+        """
+        sas_config = {
+            'input_file_group': {
+                'cslc_file_list': [
+
+                ]
+            },
+            'dynamic_ancillary_file_group': {
+            }
+        }
+        # Test uncompressed files only as input
+        cslc_file_list = ['t087_185683_iw2_20180222_VV.h5', 't087_185683_iw2_20180306_VV.h5',
+                          't087_185684_iw2_20180222_VV.h5', 't087_185684_iw2_20180306_VV.h5']
+        for f in cslc_file_list:
```

Comment:
This block from 425-427 is repeated enough that I think it merits its own helper function.